data/migrations/20210713225521_updated-table.js

```diff
@@ -0,0 +1,32 @@
+exports.up = function (knex) {
+  return knex.schema
+    .raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')
+    .createTable('incidents', (incidents) => {
+      incidents.increments('id');
+      incidents.date('date_created');
+      incidents.string('tweet_id');
+      incidents.string('user_name');
+      incidents.varchar('desc', 10000);
+      incidents.string('city').defaultTo(null);
+      incidents.string('state').defaultTo(null);
+      incidents.float('lat').defaultTo(null);
+      incidents.float('long').defaultTo(null);
+      incidents.string('title').defaultTo(null);
+      incidents.string('force_rank');
+      incidents.string('status').defaultTo('pending');
+      incidents.float('confidence');
+      incidents.string('tags');
+    })
+    .createTable('profiles', function (table) {
+      table.string('id').notNullable().unique().primary();
```

Comment:
What would determine the id? Would this be better as "increments"?

Reply:
same as before this was just the convention that was used before